web_development_101/installations/setting_up_git.md

```diff
@@ -121,6 +121,26 @@ Now, go back to GitHub in your browser window and paste the key you copied into
 
 #### Step 2.5 Testing your key
 
+First, verify that your new key is being used:
+
+~~~bash
+ssh-add -l -E md5
+~~~
+
+The output should be:
+
+~~~bash
+2048 MD5:[the signature] [your email] (RSA)
+~~~
+
+If the output is "The agent has no identities.", you need to add it. For Linux/WSL, enter:
+
+~~~bash
+echo 'ssh-add ~/.ssh/id_rsa &>/dev/null' >> ~/.bashrc
```

Comment:
I don't think we want to add this to the bashrc/bash_profile. This would attempt to add it to the authentication agent every time someone opened a terminal.  Maybe it would be better to have the ssh-add command be ran when setting up git earlier. see [here]https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agentl) . It looks like I forgot to add that while writing that lesson. Although, `~/.ssh/id_rsa` should be located by the auth agent automagically.

Reply:
The problem with this method is that it only last until you close the terminal for some reasons. That's why my method add it every time you open the terminal